tests/acceptance/02_classes/02_functions/empty_list_or_data_doesnt_return_true_for_filesexist.cf

```diff
@@ -10,10 +10,6 @@ bundle agent test
       "description"
```

Comment:
Since we are yanking the soft fail which documented the issue number I think we should add a promisee to the description.

```
"description" -> { "CFE-2631" }
```

Reply:
> Since we are yanking the soft fail which documented the issue number I think we should add a promisee to the description.

Good point. Added.